.credo.exs

```diff
@@ -0,0 +1,217 @@
+# This file contains the configuration for Credo and you are probably reading
```

Comment:
Personally, I would have not included Credo in a fix PR like this. However, as I don't very actively maintain this code I'm fine with it being added.